src/java.base/share/classes/jdk/classfile/Opcode.java

```diff
@@ -36,263 +36,262 @@
+    JSR(Classfile.JSR, 3, Kind.UNSUPPORTED),
+    RET(Classfile.RET, 2, Kind.UNSUPPORTED),
+    TABLESWITCH(Classfile.TABLESWITCH, -1, Kind.TABLE_SWITCH),
+    LOOKUPSWITCH(Classfile.LOOKUPSWITCH, -1, Kind.LOOKUP_SWITCH),
+    IRETURN(Classfile.IRETURN, 1, Kind.RETURN, TypeKind.IntType),
+    LRETURN(Classfile.LRETURN, 1, Kind.RETURN, TypeKind.LongType),
+    FRETURN(Classfile.FRETURN, 1, Kind.RETURN, TypeKind.FloatType),
+    DRETURN(Classfile.DRETURN, 1, Kind.RETURN, TypeKind.DoubleType),
+    ARETURN(Classfile.ARETURN, 1, Kind.RETURN, TypeKind.ReferenceType),
+    RETURN(Classfile.RETURN, 1, Kind.RETURN, TypeKind.VoidType),
+    GETSTATIC(Classfile.GETSTATIC, 3, Kind.FIELD_ACCESS),
+    PUTSTATIC(Classfile.PUTSTATIC, 3, Kind.FIELD_ACCESS),
+    GETFIELD(Classfile.GETFIELD, 3, Kind.FIELD_ACCESS),
+    PUTFIELD(Classfile.PUTFIELD, 3, Kind.FIELD_ACCESS),
+    INVOKEVIRTUAL(Classfile.INVOKEVIRTUAL, 3, Kind.INVOKE),
+    INVOKESPECIAL(Classfile.INVOKESPECIAL, 3, Kind.INVOKE),
+    INVOKESTATIC(Classfile.INVOKESTATIC, 3, Kind.INVOKE),
+    INVOKEINTERFACE(Classfile.INVOKEINTERFACE, 5, Kind.INVOKE),
+    INVOKEDYNAMIC(Classfile.INVOKEDYNAMIC, 5, Kind.INVOKE_DYNAMIC),
+    NEW(Classfile.NEW, 3, Kind.NEW_OBJECT),
+    NEWARRAY(Classfile.NEWARRAY, 2, Kind.NEW_PRIMITIVE_ARRAY),
+    ANEWARRAY(Classfile.ANEWARRAY, 3, Kind.NEW_REF_ARRAY),
+    ARRAYLENGTH(Classfile.ARRAYLENGTH, 1, Kind.OPERATOR, TypeKind.IntType),
+    ATHROW(Classfile.ATHROW, 1, Kind.THROW_EXCEPTION),
+    CHECKCAST(Classfile.CHECKCAST, 3, Kind.TYPE_CHECK),
+    INSTANCEOF(Classfile.INSTANCEOF, 3, Kind.TYPE_CHECK),
+    MONITORENTER(Classfile.MONITORENTER, 1, Kind.MONITOR),
+    MONITOREXIT(Classfile.MONITOREXIT, 1, Kind.MONITOR),
+    MULTIANEWARRAY(Classfile.MULTIANEWARRAY, 4, Kind.NEW_MULTI_ARRAY),
+    IFNULL(Classfile.IFNULL, 3, Kind.BRANCH, TypeKind.ReferenceType),
+    IFNONNULL(Classfile.IFNONNULL, 3, Kind.BRANCH, TypeKind.IntType),
+    GOTO_W(Classfile.GOTO_W, 5, Kind.BRANCH, TypeKind.VoidType),
+    JSR_W(Classfile.JSR_W, 5, Kind.UNSUPPORTED),
+    ILOAD_W(0xc400 | Classfile.ILOAD, 4, Kind.LOAD, TypeKind.IntType, -1),
+    LLOAD_W(0xc400 | Classfile.LLOAD, 4, Kind.LOAD, TypeKind.LongType, -1),
+    FLOAD_W(0xc400 | Classfile.FLOAD, 4, Kind.LOAD, TypeKind.FloatType, -1),
+    DLOAD_W(0xc400 | Classfile.DLOAD, 4, Kind.LOAD, TypeKind.DoubleType, -1),
+    ALOAD_W(0xc400 | Classfile.ALOAD, 4, Kind.LOAD, TypeKind.ReferenceType, -1),
+    ISTORE_W(0xc400 | Classfile.ISTORE, 4, Kind.STORE, TypeKind.IntType, -1),
+    LSTORE_W(0xc400 | Classfile.LSTORE, 4, Kind.STORE, TypeKind.LongType, -1),
+    FSTORE_W(0xc400 | Classfile.FSTORE, 4, Kind.STORE, TypeKind.FloatType, -1),
+    DSTORE_W(0xc400 | Classfile.DSTORE, 4, Kind.STORE, TypeKind.DoubleType, -1),
+    ASTORE_W(0xc400 | Classfile.ASTORE, 4, Kind.STORE, TypeKind.ReferenceType, -1),
+    RET_W(0xc400 | Classfile.RET, 4, Kind.UNSUPPORTED),
+    IINC_W(0xc400 | Classfile.IINC, 6, Kind.INCREMENT, TypeKind.IntType, -1);
```

Comment:
These can use the [`Classfile.WIDE`] constant:
```suggestion
    ILOAD_W((Classfile.WIDE << 8) | Classfile.ILOAD, 4, Kind.LOAD, TypeKind.IntType, -1),
    LLOAD_W((Classfile.WIDE << 8) | Classfile.LLOAD, 4, Kind.LOAD, TypeKind.LongType, -1),
    FLOAD_W((Classfile.WIDE << 8) | Classfile.FLOAD, 4, Kind.LOAD, TypeKind.FloatType, -1),
    DLOAD_W((Classfile.WIDE << 8) | Classfile.DLOAD, 4, Kind.LOAD, TypeKind.DoubleType, -1),
    ALOAD_W((Classfile.WIDE << 8) | Classfile.ALOAD, 4, Kind.LOAD, TypeKind.ReferenceType, -1),
    ISTORE_W((Classfile.WIDE << 8) | Classfile.ISTORE, 4, Kind.STORE, TypeKind.IntType, -1),
    LSTORE_W((Classfile.WIDE << 8) | Classfile.LSTORE, 4, Kind.STORE, TypeKind.LongType, -1),
    FSTORE_W((Classfile.WIDE << 8) | Classfile.FSTORE, 4, Kind.STORE, TypeKind.FloatType, -1),
    DSTORE_W((Classfile.WIDE << 8) | Classfile.DSTORE, 4, Kind.STORE, TypeKind.DoubleType, -1),
    ASTORE_W((Classfile.WIDE << 8) | Classfile.ASTORE, 4, Kind.STORE, TypeKind.ReferenceType, -1),
    RET_W((Classfile.WIDE << 8) | Classfile.RET, 4, Kind.UNSUPPORTED),
    IINC_W((Classfile.WIDE << 8) | Classfile.IINC, 6, Kind.INCREMENT, TypeKind.IntType, -1);
```

[`Classfile.WIDE`]: https://htmlpreview.github.io/?https://raw.githubusercontent.com/openjdk/jdk-sandbox/classfile-api-javadoc-branch/doc/classfile-api/javadoc/jdk/classfile/Classfile.html#WIDE

Reply:
fixed, thanks